h2o-py/h2o/h2o.py

```diff
@@ -993,25 +991,12 @@ def get_model(model_id):
     assert_is_type(model_id, str)
     model_json = api("GET /3/Models/%s" % model_id)["models"][0]
     algo = model_json["algo"]
-    if algo == "svd":            m = H2OSVD()
-    elif algo == "pca":          m = H2OPrincipalComponentAnalysisEstimator()
-    elif algo == "drf":          m = H2ORandomForestEstimator()
-    elif algo == "naivebayes":   m = H2ONaiveBayesEstimator()
-    elif algo == "kmeans":       m = H2OKMeansEstimator()
-    elif algo == "glrm":         m = H2OGeneralizedLowRankEstimator()
-    elif algo == "glm":          m = H2OGeneralizedLinearEstimator()
-    elif algo == "gbm":          m = H2OGradientBoostingEstimator()
-    elif algo == "deepwater":    m = H2ODeepWaterEstimator()
-    elif algo == "xgboost":      m = H2OXGBoostEstimator()
-    elif algo == "word2vec":     m = H2OWord2vecEstimator()
-    elif algo == "generic": m = H2OGenericEstimator()
-    elif algo == "deeplearning":
-        if model_json["output"]["model_category"] == "AutoEncoder":
-            m = H2OAutoEncoderEstimator()
-        else:
-            m = H2ODeepLearningEstimator()
-    elif algo == "stackedensemble": m = H2OStackedEnsembleEstimator()
-    elif algo == "isolationforest": m = H2OIsolationForestEstimator()
```

Comment:
this list was not really up-to-date (missing `psvm`, `coxph`, maybe others) + created a hard dependency on classes from submodule.
Replaced by auto-registration of estimators.

Reply:
if you want me to move those changes in another branch, it's easy: just did it here because some new tests were failing.

Reply:
LGTM, I don't see an issue - thank you